src/framework-ondata.js

```diff
@@ -456,3 +459,50 @@ function shouldRunUpdater(vni, isStale) {
     // TODO: May add different policies in the future.
     return (!isStale) && haveAllInputs(vni) && dataIsGood(vni);
 }
+
+/**
+ * Append any input state metadata to the outputState so it gets passed on to downstream nodes
```

Comment:
s/Append/Set default output metadata by appending/

Also add comment: This will be called before the updater is called, so that the updater or wrapper can modify this metadata as needed.